server/cron_service.go

```diff
@@ -0,0 +1,137 @@
+package server
+
+import (
+	"os"
+	"os/signal"
+	"strconv"
+	"syscall"
+	"time"
+
+	stats "github.com/lyft/gostats"
+	"github.com/runatlantis/atlantis/server/events"
+	"github.com/runatlantis/atlantis/server/events/models"
+	"github.com/runatlantis/atlantis/server/logging"
+)
+
+type CronService struct {
```

Comment:
I think we should rename this to `BackgroundService` or something similar, `CronService` might be confusing since we are not actually using cron here.

Reply:
Yeah thats fine, will change.

Reply:
Maybe ScheduledService

Reply:
Yeah, that sounds good.